package.json

```diff
@@ -47,7 +47,7 @@
         "db.js": "^0.15.0",
         "debug": "^4.3.2",
         "enketo-core": "5.18.1",
-        "enketo-transformer": "1.43.0",
+        "enketo-transformer": "git+https://github.com/eyelidlessness/enketo-transformer.git#bug/jr-url-space",
```

Comment:
Yes, and enketo-core as well. That's the only reason this is currently in Draft.

After all the stress of recent releases, I'd really like to find a better way to address these kinds of cross-cutting concerns. To be honest, it's not clear that the current delineation of responsibilities between transformer/core/express is ideal. But in the meantime it might be a good idea to think about a monorepo so making changes in tandem is less of a fuss.